<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>AddParamsDialog</class>
	<widget class="QDialog" name="AddParamsDialog">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>440</width>
				<height>260</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Thêm thông tin</string>
		</property>
		<layout class="QGridLayout" name="gridLayout_2">
			<item row="6" column="0">
				<layout class="QHBoxLayout" name="horizontalLayout">
					<item>
						<spacer name="horizontalSpacer">
							<property name="orientation">
								<enum>Qt::Horizontal</enum>
							</property>
							<property name="sizeHint" stdset="0">
								<size>
									<width>40</width>
									<height>20</height>
								</size>
							</property>
						</spacer>
					</item>
					<item>
						<widget class="QPushButton" name="btnAdd">
							<property name="text">
								<string>Thêm</string>
							</property>
						</widget>
					</item>
					<item>
						<spacer name="horizontalSpacer_2">
							<property name="orientation">
								<enum>Qt::Horizontal</enum>
							</property>
							<property name="sizeHint" stdset="0">
								<size>
									<width>40</width>
									<height>20</height>
								</size>
							</property>
						</spacer>
					</item>
				</layout>
			</item>
			<item row="0" column="0">
				<widget class="QLabel" name="label">
					<property name="text">
						<string>Thêm các thông tin...</string>
					</property>
				</widget>
			</item>
			<item row="1" column="0">
				<widget class="QPlainTextEdit" name="txtParamNames"/>
			</item>
			<item row="3" column="0">
				<widget class="QCheckBox" name="chxApplyAll">
					<property name="text">
						<string>Thêm vào các đối tượng khác</string>
					</property>
				</widget>
			</item>
			<item row="5" column="0">
				<spacer name="verticalSpacer">
					<property name="orientation">
						<enum>Qt::Vertical</enum>
					</property>
					<property name="sizeHint" stdset="0">
						<size>
							<width>20</width>
							<height>40</height>
						</size>
					</property>
				</spacer>
			</item>
			<item row="4" column="0">
				<widget class="QCheckBox" name="chxDefault">
					<property name="text">
						<string>Thông tin mặc định áp dụng cho tất cả các đối tượng</string>
					</property>
				</widget>
			</item>
		</layout>
	</widget>
	<resources/>
	<connections/>
</ui>
